terracumber_config/tf_files/SUSEManager-4.3-build-validation.tf

```
@@ -299,7 +299,7 @@ module "server" {
   }
 
   server_mounted_mirror = "minima-mirror-bv.mgr.prv.suse.net"
-  repository_disk_size = 1500
+  repository_disk_size = 2000
```

```suggestion
  repository_disk_size = 1700
```

Let's be more conservative.